### PR TITLE
dnsname-cni: init at 1.1.1

### PR DESCRIPTION
**Pull request by mikroskeem**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 124251` at 2d2a884d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>dnsname-cni</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/applications/networking/cluster/dnsname-cni/default.nix:13:15:
```
   |
13 |   patches = [ ./hardcode-dnsmasq-path.patch ];
   |               ^
```
</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6599,6 +6599,16 @@
...
+  mikroskeem = {
+    email = "mikroskeem@mikroskeem.eu";
+    github = "mikroskeem";
+    githubId = 3490861;
+    name = "Mark Vainomaa";
+    keys = [{
+      longkeyid = "rsa4096/0xDA015B05B5A11B22";
+      fingerprint = "DB43 2895 CF68 F0CE D4B7  EF60 DA01 5B05 B5A1 1B22";
+    }];
+  };
```
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:6611]: Will this not clash with the other PR https://github.com/NixOS/nixpkgs/pull/124248?

Comment (mikroskeem) [reply maintainers/maintainer-list.nix:6611]: It will. If one of these two get merged, I'll rebase.

Comment (mikroskeem) [reply maintainers/maintainer-list.nix:6611]: Or well, probably git is smart enough to figure out that commits are 1:1 (I just cherry picked the commit)

Comment (jonringer) [reply maintainers/maintainer-list.nix:6611]: it should be able to filter it out

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 124251 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>dnsname-cni</li>  </ul></details>

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jonringer) [reply maintainers/maintainer-list.nix:6611]: the rebase merge only committed 1 commit

**Pull request merged**